build/Targets/Settings.props

```diff
@@ -48,7 +48,7 @@
     <ToolsetCompilerPackageVersion Condition="'$(OS)' == 'Windows_NT'">$(MicrosoftNetCompilersVersion)</ToolsetCompilerPackageVersion>
     <ToolsetCompilerPackageVersion Condition="'$(OS)' != 'Windows_NT'">$(MicrosoftNETCoreCompilersVersion)</ToolsetCompilerPackageVersion>
     <ToolsetCompilerPropsFilePath>$(NuGetPackageRoot)/$(ToolsetCompilerPackageName.ToLower())/$(ToolsetCompilerPackageVersion)/build/$(ToolsetCompilerPackageName).props</ToolsetCompilerPropsFilePath>
-    <TargetFrameworkRootPath>$(NuGetPackageRoot)\RoslynTools.ReferenceAssemblies\$(RoslynToolsReferenceAssembliesVersion)\tools\Framework</TargetFrameworkRootPath>
+    <TargetFrameworkRootPath>$(NuGetPackageRoot)\roslyntools.referenceassemblies\0.1.1\tools\framework</TargetFrameworkRootPath>
```

Comment:
Is this version number hardcoded on purpose?

Reply:
No, I must have missed that when I was prototyping. I'll replace with the property.